source/components/molecules/EditableList/EditableList.js

```diff
@@ -201,30 +206,38 @@ function EditableList({
     >
       <EditableListBody>
         {inputs.map((input, index) => (
-          <EditableListItem
-            colorSchema={colorSchema}
-            editable={editable}
-            key={input.key}
-            error={error ? error[input.key] : undefined}
-            activeOpacity={1.0}
-            onPress={() => {
-              if (editable && inputRefs.current?.[index]?.focus) inputRefs.current[index].focus();
-              else if (editable && inputRefs.current?.[index]?.togglePicker)
-                inputRefs.current[index].togglePicker();
-            }}
-          >
-            <EditableListItemLabelWrapper alignAtStart={input.type === 'select'}>
-              <EditableListItemLabel>{input.label}</EditableListItemLabel>
-            </EditableListItemLabelWrapper>
-            <EditableListItemInputWrapper>
-              <InputComponent
-                {...{ input, colorSchema, editable, onChange, onInputBlur, value, state }}
-                ref={(el) => {
-                  inputRefs.current[index] = el;
-                }}
-              />
-            </EditableListItemInputWrapper>
-          </EditableListItem>
+          <>
+            <EditableListItem
+              colorSchema={colorSchema}
+              editable={editable}
+              key={input.key}
+              error={error ? error[input.key] : undefined}
+              activeOpacity={1.0}
+              onPress={() => {
+                if (editable && inputRefs.current?.[index]?.focus) inputRefs.current[index].focus();
+                else if (editable && inputRefs.current?.[index]?.togglePicker)
+                  inputRefs.current[index].togglePicker();
+              }}
```

Comment:
Personal preference but rather se this function implemented as an handler function. ie
```
handlePressListItem() {
....
}

<EditableListItem .... onPress={handlePressItem} />
...
```